### PR TITLE
Feature/duplicate access directives

### PR DESCRIPTION
**Pull request by membphis**
I hereby granted the copyright of the changes in this pull request
to the authors of this lua-nginx-module project.

### CODE REVIEW

File: t/024-access/satisfy-dup-cmds.t [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    GET /test
+--- more_headers
+--- response_body
+something important
+--- error_code: 200
+--- grep_error_log eval
+qr/access \d/
+--- grep_error_log_out
+access 1
+
```
Comment (agentzh) [inline t/024-access/satisfy-dup-cmds.t]: Let's remove this extra blank line :)

Comment (membphis) [inline t/024-access/satisfy-dup-cmds.t]: Fixed & did rebase :)

**Comment by agentzh** [comment]
@doujiang24 @moonbingbing Mind to look at this PR? Thanks!

**Comment by membphis** [comment]
do not need this feature now.

**Pull request closed without merging**